Micromouse/Direction.cpp

```diff
@@ -30,3 +30,16 @@ int DirOp::angleDiff(Direction d1, Direction d2) {
         angle += 360;
     return angle;
 }
+
+Direction DirOp::relToDir(Relation relation, Direction direction) {
```

Comment:
We could override an operator here (`operator+`?) but overriding operators everywhere could be cryptic.